tribe-customizations/v2/week/grid-body/events-row-header.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * View: Week View - Events Row Header
+ *
+ * This is a template override of the file at:
+ * events-calendar-pro/src/views/v2/week/grid-body/events-row-header.php
+ *
+ * This template override is needed to make the The Events Calendar Pro Extension: Limit Week View Time Range V2
+ * work with the updated (V2) designs.
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version 5.0.0
+ */
+?>
+<?php
+$time_format = apply_filters( 'tribe_events_week_sidebar_time_format',
+                              tribe_get_option( 'tribe_ext_limit_week_view_time_range_sidebar_time_format',
+                                                'g a' ) );
+?>
+
+<div class="tribe-events-pro-week-grid__events-row-header" role="rowheader">
+
+	<?php
+	// Set up the new header
+	for ( $i = $ext_options['grid_start_time']; $i <= $ext_options['grid_end_time']; $i++ ) {
+		$classes[] = 'tribe-events-pro-week-grid__events-time-tag';
+		$dt        = $i . ':00';
+		$label     = date( $time_format, mktime( $i, 0, 0, 1, 1, 2020 ) );
```

Comment:
we never want to use `date()` and especially not a hard-coded date, right?

Reply:
I could use some help / insight here.
This is the part responsible for the hour marks on the week view, see screenshot.
https://dl.dropbox.com/s/714dg8gpehknen0/shot_201020_192010.png

There is an option in the backend to choose a formatting for that whether it is 12h or 24h, show am/PM etc. based on date formatting settings in PHP.

(In the original code this was hard coded as '1 am ... 2 am ...'.

Here, because of the formatting option I need to have a time - an hour to be specific, the date itself doesn't matter - to be able to format it. How could I go about this better?

Let's say the `$time_format` is `g a` to get the output of '1 am ... 2 am ...' etc. where `$i` will give me the specific hour.
How can this be done better / without using a hard coded `date()`?

Reply:
This still uses `date()` but without a hardcoded one.
https://stackoverflow.com/questions/6404845/creating-select-list-in-half-hour-increments
Is this good approach?

Reply:
I used that approach.
`$label     = date( $time_format, strtotime( $dt ) );`